src/script/view_model/CallingViewModel.ts

```diff
@@ -304,7 +225,11 @@ export class CallingViewModel {
 
       const subconversation = await mlsService.getConferenceSubconversation(conversationId);
 
-      const {epoch, keyLength, secretKey, members} = await getSubconversationEpochInfo(conversationId, subconversation);
+      const {epoch, keyLength, secretKey, members} = await getSubconversationEpochInfo(
+        {mlsService},
+        conversationId,
+        subconversation,
+      );
       this.callingRepository.setEpochInfo(conversationId, {epoch, keyLength, secretKey}, members);
```

Comment:
Isn't this logic also included in the `subscribeToEpochUpdates` call? I believe, when subscribed, it also get the current version, am I wrong?